server/core/store.go

```diff
@@ -33,7 +33,7 @@ const (
 	storePersistInterval = 5 * time.Minute
 	initialMinSpace      = 8 * units.GiB // 2^33=8GB
 	slowStoreThreshold   = 80
-	awakenStoreInterval  = 30 * time.Second
+	awakenStoreInterval  = 10 * time.Minute // 2 * slowScoreRecoveryTime
```

Comment:
Do we need to test this config change?

Reply:
> Do we need to test this config change?

Yes, ut `TestAwakenStore` has contained the related testing on it. And the e-2-e testing has proved that it's valid.